jormungandr/src/network/p2p/policy.rs

```diff
@@ -116,7 +121,12 @@ impl poldercast::Policy for Policy {
     fn check(&mut self, node: &mut Node) -> PolicyReport {
         let id = node.id().to_string();
         let logger = self.logger.new(o!("id" => id));
-
+        // if the node is whitelisted
+        if let Some(Some(socket_address)) = node.address().map(|a| a.to_socketaddr()) {
+            if self.quarantine_whitelist.contains(&socket_address) {
+                return PolicyReport::None;
+            }
+        }
```

Comment:
If it is all the same I would move this inside quarantine checks https://github.com/input-output-hk/jormungandr/blob/5b6268aea9d9fa9e6662586a3ff6b840123a4835/jormungandr/src/network/p2p/policy.rs#L157-L162 and maybe add a debug log before `return` 
```rust
debug!(
    logger,
    "node is whitelisted, peer_addr: {}",
    socket_address.to_string()
);
```

Reply:
yes, in the `if else` to make things more consistent:

```
} else if node in the whitelist {
    debug!("should be quarantined, but whitelist so it's cool bro...");
    PolicyReport::None
 } else { 
     // if the record is not `clear` then we quarantine the block for some time 
     debug!(logger, "move node to quarantine"); 
     self.update(node.id().clone()); 
     PolicyReport:: LiftQuarantine; // just in case! 
 } 
```